tests/acceptance/expectUserAndAdminLoginCest.php

```diff
@@ -6,28 +6,29 @@
  */
 class expectUserAndAdminLoginCest
 {
-    public function _before(AcceptanceTester $I)
-    {
+    public function _before(AcceptanceTester $I) {
         //runs before each test
         $I->amOnPage('/');
     }
-    public function _after(AcceptanceTester $I)
-    {
+
+    public function _after(AcceptanceTester $I) {
         //runs after each test
     }
-    // tests
     
+    // tests
     public function expectsUserToLogin(AcceptanceTester $I) {
         $I->wantTo('log in as a user');
         $I->loginAs('chappress', 'password');
+        $I->cantSeeElement('#login_error');
```

Comment:
I think there's a better way to do this, but when I used `$I->amOnPage(/wp-admin/profile.php);` it still passed regardless.